packages/cdk-contentful-webhook/lambda-file-sizes.json

```diff
@@ -1 +1 @@
+[{"timestamp":1639158550913,"files":[{"filename":"contentful-webhook/index.js","previous":225902,"size":225836,"diff":-66}]},{"timestamp":1637517394655,"files":[{"filename":"contentful-webhook/index.js","previous":225899,"size":225902,"diff":3}]},{"timestamp":1633809671663,"files":[{"filename":"contentful-webhook/index.js","previous":231384,"size":225899,"diff":-5485}]},{"timestamp":1633807456780,"files":[{"filename":"contentful-webhook/index.js","previous":231381,"size":231384,"diff":3}]},{"timestamp":1620565790484,"files":[{"filename":"contentful-webhook/index.js","previous":231292,"size":231381,"diff":89}]},{"timestamp":1620265629961,"files":[{"filename":"contentful-webhook/index.js","previous":231381,"size":231292,"diff":-89}]},{"timestamp":1615923664936,"files":[{"filename":"contentful-webhook/index.js","previous":1050356,"size":231381,"diff":-818975}]},{"timestamp":1615923376037,"files":[{"filename":"contentful-webhook/index.js","previous":1047464,"size":1050356,"diff":2892}]},{"timestamp":1615809010241,"files":[{"filename":"contentful-webhook/index.js","previous":221347,"size":1047464,"diff":826117}]},{"timestamp":1615022511778,"files":[{"filename":"contentful-webhook/index.js","previous":221344,"size":221347,"diff":3}]},{"timestamp":1609970894222,"files":[{"filename":"contentful-webhook/index.js","previous":217379,"size":221344,"diff":3965}]},{"timestamp":1609970375754,"files":[{"filename":"contentful-webhook/index.js","previous":203415,"size":217379,"diff":13964}]},{"timestamp":1607360020128,"files":[{"filename":"contentful-webhook/index.js","previous":203418,"size":203415,"diff":-3}]},{"timestamp":1600966253111,"files":[{"filename":"contentful-webhook/index.js","previous":203440,"size":203418,"diff":-22}]},{"timestamp":1599246565057,"files":[{"filename":"contentful-webhook/index.js","previous":202993,"size":203440,"diff":447}]},{"timestamp":1598552652514,"files":[{"filename":"contentful-webhook/index.js","previous":202992,"size":202993,"diff":1}]},{"timestamp":1596542526761,"files":[{"filename":"contentful-webhook/index.js","previous":207102,"size":202992,"diff":-4110}]},{"timestamp":1596540034638,"files":[{"filename":"contentful-webhook/index.js","previous":206593,"size":207102,"diff":509}]},{"timestamp":1596457456615,"files":[{"filename":"contentful-webhook/index.js","previous":206357,"size":206593,"diff":236}]},{"timestamp":1596440551356,"files":[{"filename":"contentful-webhook/index.js","previous":206424,"size":206357,"diff":-67}]},{"timestamp":1596437759972,"files":[{"filename":"contentful-webhook/index.js","previous":541950,"size":206424,"diff":-335526}]},{"timestamp":1596413723535,"files":[{"filename":"contentful-webhook/index.js","previous":542071,"size":541950,"diff":-121}]},{"timestamp":1596407623809,"files":[{"filename":"contentful-webhook/index.js","previous":191169,"size":542071,"diff":350902}]}]
```

Comment:
These files were automatically updated after I ran `yarn test` locally. I assumed we might want to update this in version control but I'm not entirely sure. Please let me know if you'd like me to revert this.